### PR TITLE
PowerToys v0.33 updates and minor tweaks

### PR DESCRIPTION
**Pull request by dedavis6797**
@mattwojo please review and merge

### CODE REVIEW

**Comment by PRMerger6** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md [hunk 1]
```diff
@@ -87,13 +87,25 @@ With PowerToys v0.31 and on, there are system level actions you can now execute.
...
+With PowerToys v0.33 and on, the PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various plugins currently available. By clicking and expanding the various sections, you can customize the activation phrases used by each plugin. In addition you can select whether a plugin appears in global results, as well as set additional plugin options where available. 
+
+## Program parameters
+
+When starting a program with the PowerToys Run program plugin, you can specify parameters for the program to use by utilizing the following command formats:
+- `program -param ...`
+- `program /param ...`
+- `program --parameter ...`
+- `program -- parameter ...` (Here `--` is used as delimiter by PT Run.)
+
```
Comment (mattwojo) [inline hub/powertoys/run.md:109]: This parameter commands section feels a bit confusing to me. Could you provide an example of one of these parameter commands being used as a demonstration?

**Review by mattwojo — CHANGES_REQUESTED** [review]

File: hub/powertoys/run.md (hunk 1 above)
Comment (crutkas) [reply hub/powertoys/run.md:109]: agreed, this should be a real world example

File: hub/powertoys/install.md [hunk 2]
```diff
@@ -84,3 +84,7 @@ scoop update powertoys
 ```
 
 If you have issues when installing/updating, file an issue in the [Scoop repo on GitHub](https://github.com/lukesampson/scoop/issues).
+
+## Post Install
+
+After successfully installing PowerToys, an overview window will display with introductory guidance on each of the available utilities.
```
Comment (crutkas) [inline hub/powertoys/install.md:90]: why not add in the OOBE screen

**Comment by PRMerger18** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md [hunk 3]
```diff
@@ -87,13 +87,34 @@ With PowerToys v0.31 and on, there are system level actions you can now execute.
...
+- `program -param ...`
+- `program /param ...`
+- `program --parameter ...`
+- `program -- parameter ...` (Here `--` is used as a delimiter by PT Run.)
+
+For example, when launching Visual Studio Code, you can specify the folder to be opened with:
+
+`.Visual Studio Code -- C:\myFolder`
+
+Visual Studio Code also supports a set of [command line arguments](https://code.visualstudio.com/docs/editor/command-line), which can be utilized as parameters in PowerToys Run to, for instance, view the difference between files:
```
Comment (dedavis6797) [inline hub/powertoys/run.md]:
```suggestion
Visual Studio Code also supports a set of [command line options](https://code.visualstudio.com/docs/editor/command-line), which can be utilized as parameters in PowerToys Run to, for instance, view the difference between files:
```

**Comment by PRMerger20** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md [hunk 4]
```diff
@@ -87,13 +87,33 @@ With PowerToys v0.31 and on, there are system level actions you can now execute.
...
 
-## Indexer settings
+## Plugin manager
 
-If indexer settings are not set to cover all drives, you will receive the following warning:
+With PowerToys v0.33 and on, the PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various plugins currently available. By selecting and expanding the sections, you can customize the activation phrases used by each plugin. In addition, you can select whether a plugin appears in global results, as well as set additional plugin options where available. 
+
+## Program parameters
+
+When starting a program with the PowerToys Run program plugin, specify the program arguments using the programs expected format.
```
Comment (dedavis6797) [inline hub/powertoys/run.md]:
```suggestion
When starting a program with the PowerToys Run program plugin, specify the program arguments using the program's expected format.
```

**Review by crutkas — APPROVED** [review]

**Comment by PRMerger6** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ The following Run options are available in the PowerToys settings menu.
   | Ignore shortcuts in Fullscreen mode |  When in full-screen (F11), Run won't be engaged with the shortcut |
   | Maximum number of results |  Maximum number of results shown without scrolling |
   | Clear the previous query on launch | When launched, previous searches will not be highlighted |
-  | Disable drive detection warning | The warning, if all of your drives aren't indexed, will no longer be visible |
+  | Disable drive detection warning | The warning, if all of your drives aren't indexed, will no longer be visible. This setting is located in the Windows Search Plugin options. |
```
Comment (mattwojo) [inline hub/powertoys/run.md]: How do I find "Windows Search Plugin options"? 
I tried looking for it in PowerToys and on my Windows menu but get no results.

**Review by mattwojo — CHANGES_REQUESTED** [review]

**Comment by PRMerger10** [comment]
@dedavis6797 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/run.md (hunk 5 above)
Comment (crutkas) [reply hub/powertoys/run.md]: on my computer, searching for "search" and it just works.

image

Comment (crutkas) [reply hub/powertoys/run.md]: same with start menu

**Review by crutkas — APPROVED** [review]

File: hub/powertoys/run.md (hunk 5 above)
Comment (mattwojo) [reply hub/powertoys/run.md]: Right - we need to be sure to call things by their proper names, this should be reworded to:
"This setting is located in "Windows Search settings." 

When this window opens, I get a long list of options.. Indexing Status, Find My Files (Classic or Enhanced), Excluded Folders, Indexer Performance.. but nothing called "Plugin options" and don't see anything called "Disable drive detection warning"

Comment (mattwojo) [reply hub/powertoys/run.md]: I think the primary issue is that this entire sentence should be removed: "This setting is located in the Windows Search Plugin options." It is misleading and makes it sound like there is a "Disable drive detection warning" option that you should be able to find in a menu window called "Windows Search Plugin options", when in reality, I'm seeing that it is just a checkbox in PowerToys Run settings.

Comment (mattwojo) [reply hub/powertoys/run.md]: It already explains how to adjust indexer settings in the "Searching Windows" / Windows Search menu below.

**Comment by PRMerger7** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattwojo — APPROVED** [review]

**Pull request merged**